core/src/main/scala/org/http4s/blaze/pipeline/stages/monitors/ConnectionMonitor.scala

```diff
@@ -48,17 +48,17 @@ abstract class ConnectionMonitor {
     }
 
     def writeRequest(data: ByteBuffer): Future[Unit] = {
-      bytesOutBound(data.remaining)
+      bytesOutBound(data.remaining.toLong)
```

Comment:
Just for my own information, is auto widening of ints/floats a warning or or error with lightbendc?